lang/java.md

```diff
@@ -0,0 +1,114 @@
+
+**➤ Setting Up the Path for Windows :-**
+
+Assuming you have installed Java in *c:\Program Files\java\jdk* directory −
+
+- Right-click on 'My Computer' and select 'Properties'.
+- Click the 'Environment variables' button under the 'Advanced' tab.
+- Now, alter the 'Path' variable so that it also contains the path to the Java executable. Example, if the path is currently set to *'C:\WINDOWS\SYSTEM32'*, then change your 
+path to read *'C:\WINDOWS\SYSTEM32;c:\Program Files\java\jdk\bin'*
+
+**➤ Setting Up the Path for Linux, UNIX, Solaris, FreeBSD :-**
+
+Environment variable PATH should be set to point to where the Java binaries have been installed. Refer to your shell documentation, if you have trouble doing this.Example, if
+you use bash as your shell, then you would add the following line to the end of your *'.bashrc: export PATH = /path/to/java:$PATH'*
+
+
+## Popular Java Editors
+
+To write your Java programs, you will need a text editor, you can consider one of the following −
+
+- IntelliJ IDEA - An IDE written in Java for developing computer software is open source and can be downloaded from [here](https://www.jetbrains.com/idea/).
+- Netbeans − A Java IDE that is open-source and free which can be downloaded from [here](https://www.netbeans.org/index.html).
+- Eclipse − A Java IDE developed by the eclipse open-source community and can be downloaded from [here](https://www.eclipse.org/).
+
+
+## Some Cool projects in Java
+- [Realm](https://github.com/realm/realm-java) - Realm is a mobile database that runs directly inside phones, tablets or wearables.
+ 
+- [Universal Image Loader](https://github.com/nostra13/Android-Universal-Image-Loader) - UIL aims to provide a powerful, flexible and highly customizable instrument for image 
+loading, caching and displaying. It provides a lot of configuration options and good control over the image loading and caching process.
+
+- [Jenkins](https://github.com/jenkinsci/jenkins) - Jenkins is the leading open-source automation server. Built with Java, it provides over 1000 plugins to support automating
+virtually anything, so that humans can actually spend their time doing things machines cannot.
+
+- [Plaid](https://github.com/nickbutcher/plaid) - An Android (Java) app which provides design news and inspiration as well as being an example of implementing material design.
+
+- [Hystrix](https://github.com/Netflix/Hystrix) - Hystrix is a latency and fault tolerance library designed to isolate points of access to remote systems, services and 3rd party 
+libraries, stop cascading failure and enable resilience in complex distributed systems where failure is inevitable.
+
+- [Picasso](https://github.com/square/picasso) - A powerful image downloading and caching library for Android (Java).
+
+- [greenDAO](https://github.com/greenrobot/greenDAO) - greenDAO is a light & fast ORM solution for Android (Java) that maps objects to SQLite databases.
+
+
```

Comment:
Maybe a "thanks for reading" message? 🤔 🤷‍♂️

Reply:
@KhafiaAyyub: I think, this still needs your attention! :eyes: